kernel/nsh/builtins/info.c

```diff
@@ -0,0 +1,126 @@
+// SPDX-FileCopyrightText: CGL-KFS
+// SPDX-License-Identifier: BSD-3-Clause
+
+/* kernel/nsh/builtins/info.c
+ *
+ * Insert file description here
+ *
+ * created: 2022/12/09 - mrxx0 <chcoutur@student.42.fr>
+ * updated: 2022/12/13 - glafond- <glafond-@student.42.fr>
+ */ 
+
+#include <stdint.h>
+#include <kernel/print.h>
+#include <kernel/string.h>
+#include <kernel/kpm.h>
+
+#include "../../gdt_internal.h"
+#include "../../idt_internal.h"
+#include "registers_internal.h"
+
+extern t_gdt_entry *gdt;
+extern t_gdt_ptr gdtp;
+extern void *stack_bottom;
+extern void *stack_top;
+extern buddy_t *buddy;
+extern t_idt_entry idt[256];
+extern t_idt_ptr idtp;
+
+#define PRINT(reg, var, format) \
+	reg(var); kprintf(format, var)
+
+void info_registers()
+{
```

Comment:
Please put opening braces on the same line that the function name to be consistent with other files. (same thing for all the functions in this file).